csharp/src/Google.Protobuf/WellKnownTypes/Any.cs

```diff
@@ -119,7 +119,11 @@ public static partial class AnyReflection {
   ///       "value": "1.212s"
   ///     }
   /// </summary>
-  public sealed partial class Any : pb::IMessage<Any> {
+  public sealed partial class Any : pb::IMessage<Any>
+  #if !PROTOBUF_DISABLE_BUFFER_SERIALIZATION
```

Comment:
naming consistency: we have `GOOGLE_PROTOBUF_SUPPORT_SYSTEM_MEMORY`  define, but `PROTOBUF_DISABLE_BUFFER_SERIALIZATION`. Should we be more consistent an unify on `GOOGLE_PROTOBUF_..` prefix or `PROTOBUF_...` prefix everywhere. 
This is up for discussion.

Reply:
Changed to GOOGLE_PROTOBUF_DISABLE_BUFFER_SERIALIZATION